cmake/create_test.cmake

```diff
@@ -16,6 +16,28 @@ function(ginkgo_create_test test_name)
     add_test(NAME ${REL_BINARY_DIR}/${test_name} COMMAND ${TEST_TARGET_NAME})
 endfunction(ginkgo_create_test)
 
+function(ginkgo_create_test_link_hip test_name)
```

Comment:
I think this is better if named `ginkgo_create_hip_solver_test`.

Reply:
I would like to use more general one because it might also be used in preconditioner test

Reply:
Then maybe `ginkgo_create_hip_test_special_linkage` is better ?